authors/apps/comments/views.py

```diff
@@ -65,3 +67,72 @@ def destroy(self, request, *args, **kwargs):
 
     def perform_destroy(self, instance):
         instance.delete()
+
+class LikeComment(generics.GenericAPIView):
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    queryset = CommentLikeDislike.objects.all()
+    serializer_class = CommentLikesDislikeSerializer
+
+    def get(self, request, **kwargs):
+        """Get all like counts"""
+        comment = CommentLikeDislike.objects.filter(comment_id=kwargs['pk'])
+        serializer = self.serializer_class(comment, many=True)
+        return Response({
+                         "likes": serializer.data,
+                         "likesCount": len(serializer.data)
+                        })
+
+    def post(self, request, **kwargs):
+        """Like a comment """
+        current_user = request.user
+        user = Profile.objects.get(user=current_user)
+        user_id = user.id
+        comment_id = kwargs['pk']
+        liked_comment = CommentLikeDislike.objects.filter(like=True)
+        liked_by = CommentLikeDislike.objects.filter(user_id=user_id)
+        if liked_comment and liked_by:
+            return Response(
+                    data={"message": "You have already like the comment"},
```

Comment:
grammar please. how about "You already like this comment"

Reply:
Let me correct that